### PR TITLE
Add TPU Driver to jaxlib

### PR DESCRIPTION
**Pull request by henrytangoogle**

### CODE REVIEW

File: jax/lib/__init__.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def _check_jaxlib_version():
 
 _check_jaxlib_version()
 
-
+from jaxlib import tpu_client
```
Comment (skye) [inline jax/lib/__init__.py]: I think we need to commit the BUILD.bazel + install_xla_in_source_tree.sh first, build a new jaxlib, and then commit the rest of the changes?

File: jax/lib/xla_bridge.py [hunk 2]
```diff
@@ -44,7 +46,8 @@
                   'Enable 64-bit types to be used.')
 flags.DEFINE_string(
     'jax_xla_backend', 'xla',
-    'Default is "xla" for the XLA service directly.')
+    'Either "xla" for the XLA service directly, '
+    'or "tpu_driver" for a TPU Driver backend.')
```
Comment (skye) [inline jax/lib/xla_bridge.py]: Maybe provide some more context in this comment for what the TPU Driver is? You could just link https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/python/tpu_driver/README.md (and maybe provide a tiny bit more context there about when you would use TPU Driver, i.e. with colab)

Comment (skye) [reply jax/lib/xla_bridge.py]: This is also fine for now, consider optional

**Review by skye — APPROVED** [review]

File: jax/lib/__init__.py (hunk 1 above)
Comment (henrytansetiawan) [reply jax/lib/__init__.py]: Thanks for the suggestion @skye, I have updated the PR to push the BUILD.bazel + install_xla_in_source_tree.sh.

**Pull request merged**